lib/ProducerImpl.h

```diff
@@ -213,6 +215,7 @@ class ProducerImpl : public HandlerBase, public ProducerImplBase {
     ProducerInterceptorsPtr interceptors_;
 
     bool retryOnCreationError_;
+    std::atomic<long> producerRequestId_{0L};
```

Comment:
Should we init to something else like -1?

Reply:
Oh Yes, it's right. I will fix it.